package pl.tomskr.higienaharmonogrambackend.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalTime;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class HgShifts {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private LocalTime fullDate;
    private String year;
    private String dayOfWeek;
    private Integer monthNumber;
    private String monthName;
    private String shiftType;
    private Boolean isHoliday;
    private Boolean isWeekend;


}
